docs/using-parameter-store.md

```diff
@@ -0,0 +1,35 @@
+### Using Parameter Store for Secrets
+
+Here's an example of importing a single secret from Parameter Store.
+
+```YAML
+# example stack-args.yaml
+$imports:
+  dbPasswd: ssm:/staging/lp-webapp/dbPasswd
+  # the ssm: prefix ^ is used to ask for a ssm:parameterstore import
+
+StackName: iidy-demo
+Template: ./cfn-template.yaml
+Parameters:
+  DbPasswd: !$ dbPasswd
+  ...
+```
+
+See below for more on `$imports` and `includes` (i.e. the `!$` YAML tag).
```

Comment:
This sentence is out of place now...I think it refers to `yaml-preprocessing.md`.